app/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,15 @@
     <string name="empty_field_alert_dialog_title">Error</string>
     <string name="empty_field_alert_dialog_message">All fields are required. Please provide your</string>
     <string name="empty_field_alert_dialog_dismiss_message">Dismiss</string>
+    <string name="success_dialog_title">Success</string>
+    <string name="success_dialog_message">Your comment has been sent for approval.</string>
+    <string name="success_dialog_button">Got it</string>
+    <string name="error_dialog_title">Oops!</string>
```

Comment:
Why is this title different from the invalid email title? Having them as separate resources isn't bad, though maybe not needed, but it seems like we should be consistent.